Formula/sdformat8.rb

```diff
@@ -1,9 +1,9 @@
 class Sdformat8 < Formula
   desc "Simulation Description Format"
   homepage "http://sdformat.org"
-  url "https://bitbucket.org/osrf/sdformat/get/6ae595df61ad1327cebbbce.tar.gz"
-  version "8.0.0~pre4~1~6ae595d"
-  sha256 "839edf3e4cee565c9642e9ffc23f041356ae992fdfc7f828c2e5571d999f81a4"
+  url "https://bitbucket.org/osrf/sdformat/get/301cecc99c227bad0b783477a2c4f7004b99961e.tar.gz"
+  version "8.0.0~pre4~1~301cecc9"
```

Comment:
bump the `~1~` to `~2~`

Reply:
Bumped in https://github.com/osrf/homebrew-simulation/pull/583/commits/6af391f7f7dcb1f212a463c786f40264eccf8815 and also removed msgs to #584 so we can tackle one lib at a time.